### ISSUE TITLE
Mention functool

### ISSUE THREAD

**Issue by pydanny** [pydanny/cached-property#248]

**Comment by MT-Cash** [comment]
I've seen several `cached_property` sources:

* [functools.cached_property](https://docs.python.org/3/library/functools.html) (core python)
* Django: [django.utils.functional](https://docs.djangoproject.com/en/3.2/ref/utils/#module-django.utils.functional)
* Flask (werzeug)
* PyPI [cached_property](https://pypi.org/project/cached-property/) (this project)

I'd be pretty happy about an overview :-)

**Comment by mgorny** [comment]
Yes, please. `functools.cached_property` seems to be actually equivalent to `threaded_cached_property`, so a comparison would be nice.

**Comment by pydanny** [comment]
I'm open to a PR that makes the necessary changes to the README.

**Comment by m-aciek** [comment]
> `functools.cached_property` seems to be actually equivalent to `threaded_cached_property`, so a comparison would be nice.

FWIW in Python 3.12 `functools.cached_property` got rid of locking.
* https://github.com/python/steering-council/issues/172
* https://github.com/python/cpython/pull/101890
